jina/serve/consensus/run.go

```diff
@@ -201,6 +203,7 @@ func Run(myAddr string,
         sig := <-sigchnl
         log.Printf("Signal %v received", sig)
         grpcServer.Stop()
+        sock.Close()
```

Comment:
why is it needed if it is already defered?

Reply:
is it because the `os.Exit`?

Reply:
according to my research it should be called in sig term handling